src/components/Registration/RegistrationContainer.tsx

```diff
@@ -104,22 +100,12 @@ export const RegistrationContainer = withRouter(
         .searchPublicAdministrations({
           ...params
         })
-        .then(
-          (
-            response: ReturnType<
-              typeof BackendClient
-            >["searchPublicAdministrations"]
-          ) => {
-            return manageApiResponse(response);
-          }
-        )
-        .then(
-          (
-            responseData: ReturnType<typeof BackendClient>["createOrganization"]
-          ) => {
-            setAdministrations(responseData.administrations);
-          }
-        )
+        .then(response => {
+          return manageApiResponse(response);
```

Comment:
never explicitely type response, discriminate using conditions on the status field value